Configuration/Eras/python/Modifier_run3_scouting_nanoAOD_post2023_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+run3_scouting_nanoAOD_post2023 = cms.Modifier()
```

Comment:
please remove the intermediate `run3_scouting_nanoAOD_post2023` modifier, unless there's a citable policy to do so

Reply:
Sorry I might be completely wrong here. From [here](https://cmssdt.cern.ch/lxr/source/FWCore/ParameterSet/python/Config.py#1919), I understand that top level like ```Run3``` as ```ModifierChain``` has no method like ```toModify``` or ```toReplaceWith```. Therefore, I create ```run3_scouting_nanoAOD_post2023``` as ```Modifier``` and call ```toReplaceWith``` to replace the tables.